src/views/recipeView.js

```diff
@@ -3,7 +3,10 @@ import { Fraction } from 'fractional';
 import { renderError } from './spinner';
 console.log({ Fraction }, 'yyyy');
 const recipeContainer = document.querySelector('.recipe');
+
 export function renderRecipe(recipe) {
+  let servings = recipe.servings;
```

Comment:
why not const ?

Reply:
declared the variable outside of the function using let